eva_submission/eload_utils.py

```diff
@@ -359,3 +363,22 @@ def get_nextflow_config_flag(nextflow_config=None):
     if env_val:
         return f'-c {env_val}'
     return ''
+
+def convert_spreadsheet_to_json(metadata_xlsx, metadata_json_file_path):
+    if metadata_xlsx:
```

Comment:
This condition doesn't make a ton of sense here... this probably always gets checked outside the method, if we need anything at all, I would do
```
if not metadata_xlsx:
    return
metadata_xlsx_name = ...
```

Reply:
You're right. Didn't think that one through